src/SideEffect.tsx

```diff
@@ -54,7 +54,7 @@ export function RemoveScrollSideCar(props: IRemoveScrollEffectProps) {
   }, [props.inert, props.lockRef.current, props.shards]);
 
   const shouldCancelEvent = React.useCallback((event: WheelEvent | TouchEvent, parent: HTMLElement) => {
-    if ('touches' in event && event.touches.length === 2) {
+    if (('touches' in event && event.touches.length === 2) || (event.type === 'wheel' && event.ctrlKey)) {
```

Comment:
what `wheel` + `ctrlKey` means?
I am trying this combination and it does nothing.

Reply:
...as they enabled pinch-to-zoom gestures from multi-touch trackpad’s to be surfaced as mousewheelevent with the ctrlmodifier set to true...
- https://kenneth.io/post/detecting-multi-touch-trackpad-gestures-in-javascript
- https://github.com/theKashey/react-remove-scroll/issues/75#issuecomment-1193275548
- https://github.com/radix-ui/primitives/issues/1554#issuecomment-1993958918

This is how pinch to zoom works in desktop browsers 👍